aea/protocols/generator.py

```diff
@@ -396,6 +396,7 @@ def __init__(
         self._all_performatives = list()  # type: List[str]
         self._all_unique_contents = dict()  # type: Dict[str, str]
         self._all_custom_types = list()  # type: List[str]
+        self._custom_custom_types = dict()  # type: Dict[str, str]
```

Comment:
@5A11 It's very unclear what this variable is referring to (custom_custom?). Please use descriptive variable names, if not possible add a comment.